web/src/engine/osk/src/views/oskView.ts

```diff
@@ -49,7 +49,9 @@ export interface LegacyOSKEventMap {
   'helpclick'(obj: {});
   'resizemove'(obj: {});
   'show'(obj: {});
-  'hide'(obj: {});
+  'hide'(obj: {
+    HiddenByUser: boolean
```

Comment:
Is this meant to be `HiddenByUser` and not `hiddenByUser`?

Reply:
https://github.com/keymanapp/keyman/blob/6dafb6cedfa1524d901b7f555c1eee4338b95c0b/web/src/engine/main/osk/oskView.ts#L752-L755

Sadly, yes.